app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -451,11 +456,23 @@ export default {
         messagePayload.file = attachment.resource.file;
       }
 
+      if(this.ccEmails) {
+        messagePayload.ccEmails = this.ccEmails;
+      }
+
+      if(this.bccEmails) {
+        messagePayload.bccEmails = this.bccEmails;
+      }
+
       return messagePayload;
     },
     setFormatMode(value) {
       this.updateUISettings({ display_rich_content_editor: value });
     },
+    setCcEmails(value) {
+      this.bccEmails = value['bccEmails'];
```

Comment:
```suggestion
      this.bccEmails = value.bccEmails;
```
The variable is better written in dot notation